docs/second_factor_auth.md

```diff
@@ -16,6 +16,11 @@ This feature provides 2FA for OpenStack Keystone based on email.
      },
 ```
 
+When Keystone is expecting that user accepts 2FA challenge, the following error response is obtained: 
+```
+{"error": {"message": "Expecting Second Factor Authentication, email was sent. Please check it and click in provided link.", "code": 401, "title": "Unauthorized"}}
```

Comment:
I understand this is the payload in the response, but it would be great to specify also the status code (I guess that 401 Unauthorized)

Reply:
status is 401, like other Unathorized responses. But in this case, the diference is the message text "Expecting Second..."

Reply:
NTC